WBooks/CommentTableViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  CommentTableViewController.swift
+//  WBooks
+//
+//  Created by Nicolas Manograsso on 01/11/2018.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import Networking
+import ReactiveSwift
+import UIKit
+import WolmoCore
+
+class CommentTableViewController: UITableViewController {
+
+    var bookID : Int?
+    private let _commentRepo = NetworkingBootstrapper.shared.createUserCommentRepository()
+    private var _commentsArray = [UserComment]()
+    private let _cellSize : CGFloat = 140
+    private var _rows : Int!
+    private var _view : UIScrollView!
+    private let _width : CGFloat = -30
+    private let _heigthOfScrollView : CGFloat = 450
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UINib(nibName: "CommentTableViewCell", bundle: nil), forCellReuseIdentifier: "CommentTableViewCell")
+        setTableBackground()
+    }
+    
+    func setBookID(_ bookID : Int){
+        self.bookID = bookID
```

Comment:
this controller can't change id in the middle of being used right? you can make the constructor take the bookID then